client/src/components/MultiJurisdictionAudit/CSVForm/index.tsx

```diff
@@ -121,6 +121,12 @@ const CSVFileForm = ({
                   {processing && processing.error && (
                     <ErrorLabel>{processing.error}</ErrorLabel>
                   )}
+                  {processing && processing.completedAt && (
+                    <SuccessLabel>
+                      Upload successfully completed at{' '}
+                      {new Date(`${processing.completedAt}Z`).toLocaleString()}!
+                    </SuccessLabel>
+                  )}
```

Comment:
this shows up in the error case as well:
<img width="437" alt="Screen Shot 2020-11-17 at 9 44 31 AM" src="https://user-images.githubusercontent.com/530106/99426870-888dfe80-28b9-11eb-9fb8-72245572b9ba.png">